app/src/main/java/ar/com/wolox/android/example/ui/home/HomeView.kt

```diff
@@ -0,0 +1,3 @@
+package ar.com.wolox.android.example.ui.home
+
+interface HomeView
```

Comment:
Usually we declare the view interface inside the fragment file. something like this:
```
class HomeFragment : WolmoFragment<FragmentHomeBinding, HomePresenter>(), HomeView {
    ...
}

interface HomeView {
    ....
}
```